public/blog/2025-04-dm-vs-stat/2025-04-dm-vs-stat.md

```diff
@@ -0,0 +1,142 @@
+---
+author: "Janez Demšar"
+date: "2025-04-20"
+draft: false
+title: "Data tells stories; statistics shuts your mouth"
+thumbImage: "podgorica.jpg"
+frontPageImage: "podgorica.jpg"
+blog: ["data mining", "statistics", "climate data"]
+shortExcerpt: "A chance encounter and a workshop that wasn't"
+longExcerpt: "A chance encounter and a workshop that wasn't"
+---
+
+I want to tell you about some interesting things we did at a recent workshop. At the same time, it made me think about how we should — and should not — teach data literacy and data mining.
+
+## A chance encounter
+
+Before giving a workshop at a recent conference for Slovenian teachers, I had a chance meeting with... well, a meeting so brief that I perhaps misjudged the person I was talking to. Let us therefore, without loss of generality, assume it was an *imaginary person*, a fictional zealot, a guardian of statistical orthodoxy. You know, the kind of person who believes that the only way to the Truth leads through the sacred Ritual of forming the hypothesis, collecting the data, doing the sacred p-value dance (presumably after assuming the assumptions), rejecting the null hypothesis, and concluding with the necessary post-hocs.
+
+The thing that stayed with me, though, was their shiver of horror when I mentioned "outliers," and being berated like a schoolchild that this is something you should never, ever do. (Do — what? Say the word outlier?) I then tried to explain an example of what we plan to teach in schools and happened to mention clustering. I was told there are different clustering methods (well, I implemented many of them in Orange), but... really, "you want to teach clustering in schools"?!
```

Comment:
I then tried to explain an example of what we plan to teach --> I then tried to explain what we plan to teach